### PR TITLE
Core - ContainerStyle prop & pinch issue fix

### PR DESCRIPTION
**Pull request by cw-ajit**

### CODE REVIEW

File: src/PinchZoomPan.js [hunk 1]
```diff
@@ -295,16 +295,19 @@ export default class PinchZoomPan extends React.Component {
 
     doubleClick(event,pointerPosition) {
         const { doubleTapBehavior, maxScale, threshold, isControlledZoom, onDoubleClick } = this.props;
+        const { scale } = this.state;
         if (String(doubleTapBehavior).toLowerCase() === 'zoom' && this.state.scale * (1 + OVERZOOM_TOLERANCE) < this.props.maxScale) {
+            let curScaleValue;
             this.zoomIn(pointerPosition, this.ANIMATION_SPEED, 0.3, true);
-            onDoubleClick && onDoubleClick(event,true);
+            curScaleValue = scale < threshold ? threshold : maxScale;
```
Comment (cw-abhishek-desai) [inline src/PinchZoomPan.js:302]:
```suggestion
            const curScaleValue = scale < threshold ? threshold : maxScale;
```

File: src/PinchZoomPan.js [hunk 2]
```diff
@@ -295,16 +295,19 @@ export default class PinchZoomPan extends React.Component {
 
     doubleClick(event,pointerPosition) {
         const { doubleTapBehavior, maxScale, threshold, isControlledZoom, onDoubleClick } = this.props;
+        const { scale } = this.state;
         if (String(doubleTapBehavior).toLowerCase() === 'zoom' && this.state.scale * (1 + OVERZOOM_TOLERANCE) < this.props.maxScale) {
+            let curScaleValue;
```
Comment (cw-abhishek-desai) [inline src/PinchZoomPan.js:300]:
```suggestion

```

File: src/PinchZoomPan.js [hunk 3]
```diff
@@ -552,22 +555,23 @@ export default class PinchZoomPan extends React.Component {
...
         const touchAction = this.controlOverscrollViaCss
             ? browserPanActions(this.state) || 'none'
             : undefined;
 
-        const containerStyle = {
+        const style = {
             width: '100%', 
             height: '100%',
             overflow: 'hidden',
             touchAction: touchAction,
```
Comment (cw-abhishek-desai) [inline src/PinchZoomPan.js:569]: Will thsi override the above styels or ?

File: src/Utils.js [hunk 4]
```diff
@@ -17,10 +17,14 @@ export const getRelativePosition = ({ clientX, clientY }, relativeToElement) =>
...
-export const getPinchMidpoint = ([touch1, touch2]) => ({
-    x: (touch1.clientX + touch2.clientX) / 2,
-    y: (touch1.clientY + touch2.clientY) / 2,
-});
+export const getPinchMidpoint = ([touch1, touch2], image) => {
+    const rect = image.parentNode.getBoundingClientRect();
+
+    return {
+        x: (touch1.clientX + touch2.clientX) / 2 - rect.left,
+        y: (touch1.clientY + touch2.clientY) / 2 - rect.top,
```
Comment (cw-abhishek-desai) [inline src/Utils.js:25]:
```suggestion
        x: ((touch1.clientX + touch2.clientX) / 2) - rect.left,
        y: ((touch1.clientY + touch2.clientY) / 2) - rect.top,
```

Use wrapper whenever there can be clash between operations

**Review by cw-abhishek-desai — COMMENTED** [review]
Reviewed

File: src/PinchZoomPan.js (hunk 3 above)
Comment (cw-ajit) [reply src/PinchZoomPan.js:569]: Yes, will override the above styles.

**Review by cw-abhishek-desai — APPROVED** [review]
You are referencing master branch please change it to develop

**Review by cw-kailas — APPROVED** [review]
LGTM

**Pull request merged**